.circleci/config.yml

```diff
@@ -309,7 +309,7 @@ commands: # reusable commands with parameters
             echo "bundle exec cucumber --profile ci .circleci/no_internet.feature"
             bundle exec cucumber --profile ci .circleci/no_internet.feature
           environment: *malloc_preload
-      - log-memory-usage-periodic
+#      - log-memory-usage-periodic
```

Comment:
Just a test? Why does it bother you?

Reply:
While not impossible, you should see all processes memory usage with the `ps` command. So it should show what used memory. I assume a small program running and stopping shouldn't do that but it should show I believe :thinking: otherwise not sure how it can be figured out - what ate mem.